crates/runtime/src/flight/do_get.rs

```diff
@@ -65,20 +65,23 @@ pub(crate) async fn handle(
         }
         Command::CommandGetTableTypes(command) => flightsql::get_table_types::do_get(command),
         Command::CommandGetSqlInfo(command) => flightsql::get_sql_info::do_get(command),
-        _ => Err(Status::unimplemented("Not yet implemented")),
+        _ => {
+            let _start = metrics::track_flight_request("do_get", None);
+            Err(Status::unimplemented("Not yet implemented"))
+        }
     }
 }
 
 async fn do_get_simple(
     flight_svc: &Service,
     request: Request<Ticket>,
 ) -> Result<Response<<Service as FlightService>::DoGetStream>, Status> {
+    let start = metrics::track_flight_request("get_get", Some("sql_query"));
```

Comment:
```suggestion
    let start = metrics::track_flight_request("do_get", Some("sql_query"));
```

I assume this was a typo?

Reply:
Updated - thank you.